springwolf-bindings/springwolf-googlepubsub-binding/src/main/java/io/github/springwolf/bindings/googlepubsub/annotations/messages/GooglePubSubAsyncMessageSchema.java

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.springwolf.bindings.googlepubsub.annotations.messages;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * {@code @GooglePubSubAsyncMessageSchema} is a method-level annotation.
+ * It configures the message schema for the Google pubsub protocol.
+ * @see io.github.springwolf.asyncapi.v3.bindings.googlepubsub.GooglePubSubSchema
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(value = {ElementType.METHOD, ElementType.ANNOTATION_TYPE})
```

Comment:
I think we do not need the @Target and @Retention annotations here, because we want only want to use the annotation as part of GooglePubSubAsyncMessageBinding. This should also apply to  the GooglePubSubAsyncSchemaSetting annotation.